hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoContextUtils.java

```diff
@@ -0,0 +1,121 @@
+package com.hedera.services.usage.crypto;
+
+/*-
+ * ‌
+ * Hedera Services API Fees
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hederahashgraph.api.proto.java.CryptoAllowance;
+import com.hederahashgraph.api.proto.java.NftAllowance;
+import com.hederahashgraph.api.proto.java.TokenAllowance;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CryptoContextUtils {
+	private CryptoContextUtils() {
+		throw new IllegalStateException("Utility class");
```

Comment:
We could put a `UtilsConstructorTest` in _hapi-fees/_ like [this one](https://github.com/hashgraph/hedera-services/blob/master/hapi-utils/src/test/java/com/hedera/services/UtilsConstructorTest.java) for the _hapi-utils/_ module.

Reply:
Perfect ! Added a UtilsConstructorTest in hapi-fees.